app/views/dashboard/index.html.erb

```diff
@@ -73,6 +73,7 @@
             <div class="box-body text-center float-center">
               <%= new_button_to new_distribution_path, { text: "New Distribution" } %>
               <h3 class="text-center"><%= total_distributed %> items distributed <%= display_interval %></h3>
+              <h4 class="text-center">(<%= future_distributed %> items scheduled for future distribution)</h4>
```

Comment:
This is a new bit of clarification so that you can see already-scheduled (and thus not counted toward the total on-hand) distributions.